pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Common.java

```diff
@@ -2,8 +2,12 @@
 
 import com.hedera.pbj.compiler.impl.grammar.Protobuf3Parser;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Incorrect annotation (we do not use the jetbrains annotations).  Might be worth adjusting IDE settings to prefer the correct annotation (one line above).

Reply:
Oops. Removed.